tools/scripts/unit_test_coverage.sh

```diff
@@ -97,7 +97,11 @@ do
     # Delete gcda files that did not change.
     find . -name "*.gcda" | xargs md5sum | sort -k2,2 > current
     diff baseline current | grep '^<' | cut -d' ' -f4 > files_to_keep
-    find . -name "*.gcda" | grep -v -F -f files_to_keep | xargs rm
+    find . -name "*.gcda" | grep -v -F -f files_to_keep > files_to_remove
+    if [ -s files_to_remove ]
+    then
+        cat files_to_remove | xargs rm
+    fi
```

Comment:
remove `files_to_remove` once you're done with it?

Reply:
When I ran the script it left these files:
baseline
current
files_to_keep
files_to_remove